apps/laboratory/src/components/Ethers/EthersSendCallsTest.tsx

```diff
@@ -1,27 +1,61 @@
+import {
+  Box,
+  Link,
+  Stat,
+  StatLabel,
+  StatNumber,
+  StatHelpText,
+  Card,
+  CardBody
+} from '@chakra-ui/react'
+import { useCallback, useState, useEffect } from 'react'
 import { Button, Stack, Text, Spacer, Heading } from '@chakra-ui/react'
-import { useState, useEffect } from 'react'
 import { useAppKitAccount, useAppKitNetwork, useAppKitProvider } from '@reown/appkit/react'
 import { UniversalProvider } from '@walletconnect/universal-provider'
 import { useChakraToast } from '../Toast'
-import type { Address } from 'viem'
+import { parseGwei, type Address } from 'viem'
 import { vitalikEthAddress } from '../../utils/DataUtil'
 import { BrowserProvider, type Eip1193Provider } from 'ethers'
 import {
   EIP_5792_RPC_METHODS,
   WALLET_CAPABILITIES,
   getCapabilitySupportedChainInfo
 } from '../../utils/EIP5792Utils'
+import { AddIcon, DeleteIcon } from '@chakra-ui/icons'
+import { AddTransactionModal } from '../AddTransactionModal'
 import { W3mFrameProvider } from '@reown/appkit-wallet'
 type Provider = W3mFrameProvider | Awaited<ReturnType<(typeof UniversalProvider)['init']>>
 
-export function EthersSendCallsTest() {
+export function EthersSendCallsTest(params: { onCallsHash: (hash: string) => void }) {
+  const { onCallsHash } = params
```

Comment:
Nit and it's more of a general pattern but same here, can just destructure from props instead of assigning a variable